spec/models/spree/product_spec.rb

```diff
@@ -711,26 +711,6 @@ module Spree
         e.variants(true).should be_empty
       end
     end
-
-    describe '#on_hand' do
-      let(:product) { create(:product) }
-
-      context 'when the product has variants' do
-        before { create(:variant, product: product) }
-
-        it 'returns the sum of the on_hand of its variants' do
-          expect(product.on_hand).to eq(Float::INFINITY)
-        end
-      end
-
-      context 'when the product has no variants' do
-        before { product.variants.destroy_all }
-
-        it 'returns the on_hand of the master' do
-          expect(product.on_hand).to eq(product.master.on_hand)
-        end
-      end
```

Comment:
Are these cases covered elsewhere?

Reply:
yes, it's mentioned in the commit message: product_stock_spec is doing exactly the same